bin/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env sh
+
+# Synopsis:
+# Run the test runner on a solution.
```

Comment:
Pedantically, this isn't running the test runner. The 8th track actually uses it's own test runner, and frankly it's a PITA: PRs from forks can't run tests, only PRs from branches in the 8th repo can.

Reply:
Too be fair, that is only because the 8th test runner is special in that the binary is encrypted and thus CI requires access to a secret (which forks don't have access to). 

You could run the sqlite test runner and things should work fine on forks. I think a couple of tracks do this, including Unison. That said, most tracks don't, and that is usually fine as test runners should have smoke tests.

Reply:
I first thought to pull the test runner from its repository, but that is overkill. I don't care for the results.json, I only need to see if any tests fail, so the CI gets an alarm. 

I redid that part and amended the synopsis.